api/v1alpha/src/procurements/index.js

```diff
@@ -145,4 +148,23 @@ procurements.post('/projects/:projectId/procurements/approve', async (req, res)
     });
 });
 
+procurements.post('/projects/:projectId/procurements/myProducts', async (req, res) => {
```

Comment:
Should this be a custom HTTP POST resource for the redirect e.g. `procurements:products` or `procurements:myproducts`, similar to the custom method for [`accounts:register`](https://github.com/GoogleCloudPlatform/datashare-toolkit/blob/60dd081e0ee0af19dc460bcc26cd44cdf1ca01dc/api/v1alpha/src/accounts/index.js#L638) ?

Reply:
good point